### PR TITLE
revert paritytech/polkadot#6577 & related changes

### PR DESCRIPTION
**Pull request by Lederstrumpf**
Moves BEEFY pallets - in particular `pallet_mmr` - back behind `pallet_session` to address #160.

See paritytech/polkadot-sdk#3108 for the equivalent Rococo change.

### CODE REVIEW

**Review by acatangiu — APPROVED** [review]

**Review by andresilva — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

File: relay/kusama/src/lib.rs [hunk 1]
```diff
@@ -1637,6 +1630,13 @@ construct_runtime! {
 
 		// Asset rate.
 		AssetRate: pallet_asset_rate::{Pallet, Call, Storage, Event<T>} = 101,
+
+		// BEEFY Bridges support.
+		Beefy: pallet_beefy::{Pallet, Call, Storage, Config<T>, ValidateUnsigned} = 200,
+		// MMR leaf construction must be after session in order to have a leaf's next_auth_set
+		// refer to block<N>. See issue #160 for details.
+		Mmr: pallet_mmr = 201,
+		MmrLeaf: pallet_beefy_mmr = 202,
```
Comment (bkchr) [inline relay/kusama/src/lib.rs:1639]:
```suggestion
		BeefyMmrLeaf: pallet_beefy_mmr = 202,
```

File: relay/polkadot/src/lib.rs [hunk 2]
```diff
@@ -1645,6 +1638,13 @@ construct_runtime! {
 
 		// Asset rate.
 		AssetRate: pallet_asset_rate::{Pallet, Call, Storage, Event<T>} = 101,
+
+		// BEEFY Bridges support.
+		Beefy: pallet_beefy::{Pallet, Call, Storage, Config<T>, ValidateUnsigned} = 200,
+		// MMR leaf construction must be after session in order to have a leaf's next_auth_set
+		// refer to block<N>. See issue #160 for details.
+		Mmr: pallet_mmr = 201,
+		MmrLeaf: pallet_beefy_mmr = 202,
```
Comment (bkchr) [inline relay/polkadot/src/lib.rs:1647]:
```suggestion
		BeefyMmrLeaf: pallet_beefy_mmr = 202,
```

**Review by bkchr — COMMENTED** [review]
You can not just rename the pallets as otherwise their storage key will change!

**Comment by acatangiu** [comment]
> You can not just rename the pallets as otherwise their storage key will change!

Damn good catch sir! My eyes missed that completely!

**Comment by bkchr** [comment]
> > You can not just rename the pallets as otherwise their storage key will change!
> 
> Damn good catch sir! My eyes missed that completely!

I also missed that :see_no_evil:  I only have seen this because of CI failing :see_no_evil:

**Pull request merged**